tests/test_encryption.py

```
@@ -1,13 +1,18 @@
 
 def test_decrypt():
     passphrase = "passphrase"
     cleartext = "hello"
     encrypted = "$aes-256-cbc/pbkdf2-sha1$i=75000$obSTW6ittQvTtdAxonQKIw==$g3QFAC9CtBcPxoKlouqsyQ=="
-    encryptor = AESEncryptor(passphrase, iterations=75000)
+    # A função Encryptor usa 'passphrase' como primeiro argumento.
+    # Iterations são lidas do payload para decrypt, mas o encryptor precisa ser inicializado.
+    # Para o decrypt funcionar com o payload dado, o encryptor precisa ser inicializado com as iterações do payload.
+    # No entanto, a função Encryptor tem um default, então apenas passphrase é necessário para instanciar.
+    # O método decrypt da classe AESEncryptor usa as iterações do payload.
+    encryptor = Encryptor(passphrase=passphrase) # Passando passphrase
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)